src/LinuxTracing/TracerImpl.h

```diff
@@ -143,6 +143,7 @@ class TracerImpl : public Tracer {
   uint16_t stack_dump_size_;
   orbit_grpc_protos::CaptureOptions::UnwindingMethod unwinding_method_;
   std::vector<Function> instrumented_functions_;
+  std::map<uint64_t, uint64_t> absolute_address_to_size_of_functions_to_stop_at_;
```

Comment:
Similarly to the proto, I think this should be `absolute_address_to_size_of_functions_to_stop_unwinding_at_` because "stop" alone is not clear in this context. I know the name is very long, but is also very precise which is good.

Reply:
Done.